domains/tobez.json

```diff
@@ -5,6 +5,7 @@
   },
 
   "record": {
-    "URL": "https://tobez.dev"
+    "URL": "https://tobez.dev",
+    "TXT": "dh=64624183b9e763aa7a812ddf39b0ae9261491182"
```

Comment:
Not the way to do Discord TXT verification, please check our guide here https://www.is-a.dev/docs/discord_verification/